pkg/model/pki.go

```diff
@@ -45,6 +45,15 @@ func (b *PKIModelBuilder) Build(c *fi.CloudupModelBuilderContext) error {
 	}
 	c.AddTask(defaultCA)
 
+	kopsControllerBootstrapJWT := &fitasks.Keypair{
+		Name:      fi.PtrTo("kops-controller-bootstrap"),
+		Lifecycle: b.Lifecycle,
+		Subject:   "cn=kops-controller",
+		Type:      "client",
+		Signer:    defaultCA,
```

Comment:
```suggestion
		Type:      "ca",
```

This would probably be why it didn't work.